streams/src/main/java/org/apache/kafka/streams/kstream/internals/KStreamImpl.java

```diff
@@ -110,6 +116,8 @@
 
     private static final String FOREACH_NAME = "KSTREAM-FOREACH-";
 
+    private static final String TO_KTABLE_NAME = "KSTREAM-TO-KTABLE-";
```

Comment:
Nit: in `KTableImpl` the corresponding name is `KTABLE-TOSTREAM-` hence, we should use `KSTREAM-TOTABLE-` here